usr/src/uts/common/fs/zfs/sys/uberblock_impl.h

```diff
@@ -54,6 +54,11 @@ struct uberblock {
 
 	/* highest SPA_VERSION supported by software that wrote this txg */
 	uint64_t	ub_software_version;
+
+	/* These fields are reserved for features that are under development: */
+	uint64_t	ub_mmp_magic;
+	uint64_t	ub_mmp_delay;
```

Comment:
In order in avoid a potential future conflict let's reserve one additional MMP field for a sequence number.  This is something we've talked about using in the past since the existing timestamp granularity is measured in seconds.  I'll add this reserved field to ZoL MMP patch.

```c
uint64_t ub_mmp_seq;
```